packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -145,18 +145,19 @@ module.exports = {
             options: {
               formatter: eslintFormatter,
               eslintPath: require.resolve('eslint'),
-              // @remove-on-eject-begin
               baseConfig: {
                 extends: [require.resolve('eslint-config-react-app')],
               },
+              // @remove-on-eject-begin
               ignore: false,
               useEslintrc: false,
               // @remove-on-eject-end
             },
             loader: require.resolve('eslint-loader'),
           },
         ],
-        include: paths.appSrc,
+        include: paths.srcPaths,
+        exclude: [/[/\\\\]node_modules[/\\\\]/],
```

Comment:
Could you please explain why this has become necessary? Is this because non-CRA packages have their source code at the same level of nesting as their own `node_modules` and we want to skip those?

Reply:
Yep, since srcPaths includes <comp1> root (or or top-level monorepo root, depending on decision above), both  <comp1>/file.js and <comp1>/node_modules/somedep/somefile.js get included in the include filter, so we then filter out node_modules.  (Originally, only <app>/src was included, so there was no need to filter out <app>/node_modules.)

Reply:
These webpack excludes just exclude them from getting transpiled and linted.